### PR TITLE
[#722] Add AKHQ as optional tool

### PR DESCRIPTION
**Pull request by ljupcovangelski**
AKHQ is deployed if `tools.akhq.enabled=true` in the `airy.yaml` file.
Basic authentication is enabled. Default username is `admin`. The password can be retrieved from a configMap:
```
kubectl get configmap akhq-config -o jsonpath="{.data.password}"
```

### CODE REVIEW

File: infrastructure/helm-chart/values.yaml [hunk 1]
```diff
@@ -8,3 +8,4 @@ global:
     webhooksHost: "webhooks.airy"
     uiHost: "ui.airy"
     chatpluginHost: "chatplugin.airy"
+    toolsHost: "tools.airy"
```
Comment (ljupcovangelski) [inline infrastructure/helm-chart/values.yaml:11]: Akhq will be deployed and accessible through akhq.{toolsHost} or `akhq.tools.airy`.
I didn't add the DNS entry for the Vagrantbox, as `/etc/hosts` doesn't accept wildcard certificates.

Open to suggestions for this, maybe one option would be to do `tools.airy/akhq`, but this doesn't work out of the box and some tweak would be needed for traefik to strip the `/akhq` prefix, before the request is sent to the app.

**Review by pascal-airy — APPROVED** [review]
glad you made the auth work :)

File: infrastructure/airy.tpl.yaml [hunk 2]
```diff
@@ -56,3 +56,6 @@ apps:
         bucket: "changeme"
         region: "changeme"
         path: "path"
+tools:
+  akhq:
+    enabled: true
```
Comment (lucapette) [inline infrastructure/airy.tpl.yaml:61]: I wouldn't have this enabled by default

**Review by lucapette — CHANGES_REQUESTED** [review]
I like it! but it's strange that it's an optional component and it's enabled by default.

We should also provide some docs about it somewhere but I suppose that can be done in the #846 

We do need a followup ticket to make this available in the vagrant instance though

**Review by lucapette — APPROVED** [review]
:shipit:

**Pull request merged**